portfolio/src/main/webapp/index.html

```diff
@@ -8,10 +8,18 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <h1>Jimmy Coleman</h1>
+      <img src="images/jimmy-2.jpg" width="400" height="600"/>
```

Comment:
Specifying both width and height is a little dangerous because it might cause the image to stretch if the aspect ratio isn't exactly right. I'm sure you got it right here, but if you ever changed the image file it could be a problem. If you just specify width, the browser will automatically pick the correct height.